kotlin/scene-layer/README.metadata.json

```
@@ -0,0 +1,24 @@
+{
+    "category": "Layers",
+    "description": "Add a scene layer to a scene.",
+    "ignore": false,
+    "images": [
+        "scene-layer.png"
+    ],
+    "keywords": [
+        "Layers",
+        "ArcGISScene",
+        "SceneView",
+        "SceneLayer"
+    ],
+    "redirect_from": "",
```

we require a url in here for the developers website to redirect to, since this is already an existing sample on the website (see https://developers.arcgis.com/android/latest/java/sample-code/scene-layer/). 
```suggestion
    "redirect_from": "/android/latest/sample-code/scene-layer.htm",
```